x-pack/plugin/esql/qa/server/single-node/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/single_node/TSDBRestEsqlIT.java

```diff
@@ -44,9 +43,10 @@ protected String getTestRestCluster() {
     }
 
     public void testTimeSeriesQuerying() throws IOException {
-        assumeTrue("time series querying relies on query pragma", Build.current().isSnapshot());
+        assumeTrue("time series requires metrics command", Build.current().isSnapshot());
         var settings = Settings.builder()
             .loadFromStream("tsdb-settings.json", TSDBRestEsqlIT.class.getResourceAsStream("/tsdb-settings.json"), false)
+            .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
```

Comment:
Do we need this setting? Wondering if it might not cause issues if we'll want to run the test in Serverless.

Reply:
Yes, this setting ensures that we always run this test with a single shard, allowing it to pass with serverless.